cmatrix.c

```diff
@@ -87,6 +87,12 @@ int *updates = NULL; /* What does this do again? */
 volatile sig_atomic_t signal_status = 0; /* Indicates a caught signal */
 #endif
 
+#ifdef _WIN32
+char *DELIMITER = "\r\n";
```

Comment:
> `\r\n` is only used on Windows Notepad, the DOS command line, most of the Windows API, and in some (older) Windows apps.

I doubt modern text editors follow this same delimiter to separate lines. Can we instead check for both? (ie, if we have \r\n okay, but if it's just \n that's okay as well?

Reply:
Good idea, I changed it to try both cases.